Controller/PayumController.php

```diff
@@ -8,15 +8,19 @@
 
 abstract class PayumController extends AbstractController
 {
-    /**
-     * @return Payum
-     */
-    protected function getPayum()
+    protected Payum $payum;
+
+    public function __construct(Payum $payum)
+    {
+        $this->payum = $payum;
+    }
+
+    protected function getPayum(): Payum
     {
-        return $this->get('payum');
+        return $this->payum;
     }
 
-    public static function getSubscribedServices()
+    public static function getSubscribedServices(): array
```

Comment:
This will also be a BC break, the return type needs to be removed here

Reply:
ok, i will revert to 
`/**
     * @return string[]
     */
    public static function getSubscribedServices()`